src/cmake/macros/SDL2_Frontend.cmake

```diff
@@ -1,18 +1,62 @@
 MACRO(CONFIGURE_SDL2_FRONTEND _NAME_TARGET)
 
-    INCLUDE(FindPkgConfig)
+    FIND_PACKAGE(PkgConfig REQUIRED)
 
-    PKG_SEARCH_MODULE(SDL2 sdl2)
```

Comment:
According to https://cmake.org/cmake/help/latest/module/FindPkgConfig.html , IMPORTED_TARGET requires 3.7 or later so the minimum version specified on the first line of CMakeLists.txt should be bumped from 3.5 to 3.7.

Reply:
Done